src/Jackalope/Security/Privilege.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Jackalope\Security;
+
+use PHPCR\Security\PrivilegeInterface;
+
+/**
+ * {@inheritDoc}
+ */
+class Privilege implements PrivilegeInterface
+{
+    /**
+     * @var $name
+     */
+    private $name;
+
+    /**
+     * @var PrivilegeInterface[]
+     */
+    private $declaredAggregate;
+
+    /**
+     * @var PrivilegeInterface[]
+     */
+    private $aggregate = array();
+
+    public function __construct($name, $declaredAggregate = array())
+    {
+        $this->name = $name;
+        $this->declaredAggregate = $declaredAggregate;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isAbstract()
+    {
+        return false;
```

Comment:
we need to make this dynamic, since it is possible to set a privilege as abstract in Jackrabbit